.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build-and-push-docker-image:
```

Comment:
I think we can just call the job `docker` to keep it short.

Reply:
I updated it :) 22d626e